app/src/main/java/org/merakilearn/ui/onboarding/LoginViewModel.kt

```diff
@@ -18,4 +18,12 @@ class LoginViewModel(private val applicationRepo: ApplicationRepo) : ViewModel()
     fun logOut() = liveData {
         emit(applicationRepo.logOut())
     }
+
+    fun deleteFile(first: String) = liveData {
```

Comment:
as previously discussed, returning live data from a function is not a good strategy here